### DISCUSSION TITLE
Open Admin UI for regular users

### DISCUSSION THREAD

**Discussion by TomTom101** [pocketbase/pocketbase#5788] [Polls]
In order for regular users to make edits to records, I need to build a separate web app that offers login, listing of collections, selecting, editing, saving etc. All of which the beautiful built-in admin interface already has.

Wouldn't it be nice if regular users could login to the provided UI and use it to make changes? Obviously sans the admin settings, API rules (which would be applied to the UI of course)?

That would make building web frontends for the sake of providing basic CRUD operations entirely obsolete. What do you think?

**Comment by TokenChingy** [comment]
You would be effectively building a headless CMS. Pocketbase is not a CMS (in my opinion). There are plenty of other options available for this, such as [Strapi](https://strapi.io/) or [Directus](https://directus.io/solutions/headless-cms) just to name two.

**Reply by TomTom101** [reply]
Well, the admin interface partly makes it a CMS, which is great imo! I find the restriction to only make it available for admins quite unnecessary though. Strapi and Directus come with their own database as far as I can tell, they're are not the "missing CMS" on top of pocketbase, which I would like to use. Maybe pocketbase is the wrong choice then to start with, if I want users to edit records in a web interface.

**Reply by TokenChingy** [reply]
Pocketbase also has its own database, and that's SQLite. Strapi can also use SQLite if that's what is required.

In addition, Strapi can do AuthN and AuthZ, it has auto generated REST and GraphQL for its tables, you can extend it with custom routes and middleware, as well as all the sorts of other things you would expect in Pocketbase (effectively, it can be used as a framework), it has document management, it can also support plugins etc. You can also setup Strapi so certain roles can only do certain things and view certain things in the Admin UI.

I use Strapi as an example as I'm most familiar with it.

**Comment by alucab** [comment]
I think it would be amazing
I like the CRUD interface of teh Admin UI and it would be very suitable for my small application
